tensorflow/stream_executor/cuda/cuda_gpu_executor.cc

```diff
@@ -1129,6 +1129,30 @@ DeviceDescription *CUDAExecutor::PopulateDeviceDescription() const {
           CU_DEVICE_ATTRIBUTE_MAX_REGISTERS_PER_MULTIPROCESSOR, device_)
           .ValueOrDie());
 
+  const char* blank_ptx =
+      ".version 6.0\n"
+      ".target sm_30\n"
+      ".address_size 64\n"
+      "\n"
+      "        // .globl       _Z6ValAddPf\n"
+      ".visible .entry _Z6ValAddPf(\n"
+      ")\n"
+      "{\n"
+      "        ret;\n"
+      "}\n";
+  const char* kernel_name = "_Z6ValAddPf";
+
+  CUmodule blank_module;
+  CUfunction blank_function;
+  CUDADriver::LoadPtx(context_, blank_ptx, &blank_module);
+  CUDADriver::GetModuleFunction(context_, blank_module, kernel_name,
+                                &blank_function);
+
+  int bpc;
+  CUresult result = cuOccupancyMaxActiveBlocksPerMultiprocessor(
+      &bpc, blank_function, 1, (size_t)1);
```

Comment:
Is the size_t cast necessary?  If it is, we'd prefer to spell it `size_t{1}`.  The C++ gurus tell me this is the safest way to cast a constant integer to a specific type, as it will error out if the number is out of range.

Reply:
Doesn't appear necessary, must have overlooked it
```suggestion
      &bpc, blank_function, 1, 1);
```